website/pages/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -5,4 +5,204 @@ sidebar_title: Terminating Gateways
 description: Configuring Terminating Gateways on Kubernetes
 ---
 
-# Terminating Gateways on Kubernetes
\ No newline at end of file
+# Terminating Gateways on Kubernetes
+
+-> 1.8.0+: This feature is available in Consul versions 1.8.0 and higher
+
+-> 0.16.0+: This feature is available in consul-k8s versions 0.16.0 and higher
+
+~> This topic requires familiarity with [Terminating Gateways](/docs/connect/terminating-gateway).
+
+Terminating gateways are a new feature included in Consul 1.8. The correlating consul-k8s binary version is
+[0.16.0](https://github.com/hashicorp/consul-k8s/blob/master/CHANGELOG.md#0160-june-17-2020), and is required to enable
+terminating gateways. If you are using the latest official [consul-helm chart](https://github.com/hashicorp/consul-helm),
+and have not customized the [imageK8S](/docs/k8s/helm#v-global-imagek8s) configuration for any of
+your components, you should be running a compatible version by default.
+
+Adding a terminating gateway is a multi-step process:
+
+* Update the helm chart with terminating gateway config options
+* Deploying the helm chart
+* Accessing the Consul agent
+* Register external services with Consul
+
+## Update the helm chart with terminating gateway config options
+
+Minimum required Helm options:
+
+```shell
+global:
+  name: consul
+connectInject:
+  enabled: true
+terminatingGateways:
+  enabled: true
+```
+
+## Deploying the helm chart
+
+Ensure you have the latest consul-helm chart and install Consul via helm using the following
+[guide](/docs/k8s/installation/overview#installing-consul) while being sure to provide the yaml configuration
+as previously discussed.
+
+## Accessing the Consul agent
+
+You can access the Consul server directly from your host via `kubectl port-forward`, this is helpful for interacting with your Consul UI locally as well as to validate connectivity of the application.
+
+```shell-session
+$ kubectl port-foward consul-server-0 8500 &
+```
+If TLS is enabled use port 8501:
+```shell-session
+$ kubectl port-foward consul-server-0 8501 &
+```
+
+-> Be sure the latest consul binary is installed locally on your host.
+[https://releases.hashicorp.com/consul/](https://releases.hashicorp.com/consul/)
+
+If TLS is enabled set:
+```shell-session
+$ export CONSUL_HTTP_ADDR=https://localhost:8501
+```
+If ACLs are enabled also set:
+```shell-session
+$ export CONSUL_HTTP_TOKEN=$(kubectl get secret consul-bootstrap-acl-token -o jsonpath={.data.token} | base64 -D)
+$ export CONSUL_HTTP_SSL_VERIFY=false
+```
+
+## Register external services with Consul
+
+Registering the external services with Consul is a multi-step process:
+* Register external services with Consul
+* Update the terminating gateway ACL token if ACLs are enabled
+* Create the configuration entry for the terminating gateway
+* Create intentions to allow access from services in the mesh to external service
+* Define upstream annotations for any services that need to talk to the external services
```

Comment:
Should this be a numbered list instead?

```suggestion
1. Register external services with Consul
1. Update the terminating gateway ACL token if ACLs are enabled
1. Create the configuration entry for the terminating gateway
1. Create intentions to allow access from services in the mesh to external service
1. Define upstream annotations for any services that need to talk to the external services
```

Markdown should render this as 1 - 5.